typescript/sdk/src/consts/multisigIsm.ts

```diff
@@ -401,6 +401,11 @@ export const defaultMultisigConfigs: ChainMap<MultisigConfig> = {
     ],
   },
 
+  oortmainnet: {
+    threshold: 1,
+    validators: ['0x9b7ff56cd9aa69006f73f1c5b8c63390c706a5d7'],
+  },
```

Comment:
These have always been in the SDK IIRC. I considered moving them to the registry but I believe @tkporter suggested they stay here. Is it time to revisit that? Seems like reducing the package touch points from 3 (registry, infra, SDK), to 2 would be a good thing

Reply:
Yeah I think there have been a few recent discussions about moving this into the registry, unfortunately just hasn't been prioritized yet